lib/sidekiq/web/application.rb

```diff
@@ -291,6 +294,30 @@ def call(env)
       resp
     end
 
+    def get_csp_header(scheme)
+      @csp_headers ||= {}
```

Comment:
I'd make this hash a constant; I'm not sure of the lifetime of instance variables here.